### ISSUE TITLE
Reply feature does not work if the posting is a link

### ISSUE THREAD

**Issue by gakudva** [signalapp/Signal-iOS#3420]
- [x] I have searched open and closed issues for duplicates
----------------------------------------

### Bug description
Thanks to the new reply feature.  However, the reply feature does not work if the text is a link.  There is no way to reply to a post thats a link.

### Steps to reproduce
- Post a Link (for e.g., Google.com)
- To reply -> click on text to get the pop-up menu 

**Actual result:** There is no pop-up menu; the click on the link will always navigate to the link.
**Expected result:** The link text must have a place holder (just like the one you get for images) where we can get the menu

### Device info
**Device**: iOS 8+
**iOS version**:  11.3.1
**Signal version:** 2.24.1.0

**Comment by leonardoporpora** [comment]
Yes i can confirm this could eb easily fixed adding a method for quick reply. What would i do is to answer to a message not only clicking on it and than clicking reply but simply swiping left and replying to it. So if it's or not a link this wouldn't create problems

**Comment by tzs1** [comment]
I think this has been [fixed](https://github.com/signalapp/Signal-iOS/commit/0b638f4831a7001c5757b998901c5b3f40e9df9b)

**Comment by Mihahn** [comment]
Can't reproduce, is this still an issue? If not, please close your issue.
